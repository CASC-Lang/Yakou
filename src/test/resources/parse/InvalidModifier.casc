ovrd class InvalidModifier

impl InvalidModifier {
    priv mut ovrd comp {

    }

    mut ovrd new() {}
}