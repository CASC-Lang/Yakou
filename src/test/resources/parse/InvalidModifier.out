error: Cannot declare `ovrd` keyword in current context
--> ./InvalidModifier.casc:1:0
1 | ovrd class InvalidModifier
    ^^^^= hint: Remove this `ovrd` keyword
2 |
error: Cannot declare `ovrd` keyword after `mut` keyword
--> ./InvalidModifier.casc:4:9
3 | impl InvalidModifier {
4 |     priv mut ovrd comp {
             ^^^= hint: `mut` keyword here
5 |
error: Cannot declare companion block with access modifier `priv`
--> ./InvalidModifier.casc:4:4
3 | impl InvalidModifier {
4 |     priv mut ovrd comp {
        ^^^^= hint: Remove this access modifier `priv`
5 |
error: Cannot declare companion block with `ovrd` keyword
--> ./InvalidModifier.casc:4:13
3 | impl InvalidModifier {
4 |     priv mut ovrd comp {
                 ^^^^= hint: Remove this `ovrd` keyword
5 |
error: Cannot declare companion block with `mut` keyword
--> ./InvalidModifier.casc:4:9
3 | impl InvalidModifier {
4 |     priv mut ovrd comp {
             ^^^= hint: Remove this `mut` keyword
5 |
error: Cannot declare `ovrd` keyword after `mut` keyword
--> ./InvalidModifier.casc:8:4
7 |
8 |     mut ovrd new() {}
        ^^^= hint: `mut` keyword here
9 |
error: Cannot declare constructor with `ovrd` keyword
--> ./InvalidModifier.casc:8:8
7 |
8 |     mut ovrd new() {}
            ^^^^= hint: Remove this `ovrd` keyword
9 |
error: Cannot declare constructor with `mut` keyword
--> ./InvalidModifier.casc:8:4
7 |
8 |     mut ovrd new() {}
        ^^^= hint: Remove this `mut` keyword
9 |
error: Companion function cannot be mutable
--> ./InvalidModifier.casc:10:4
9  |
10 |     mut fn a1() {}
         ^^^= hint: Remove this `mut` keyword
11 | }